crates/chia-consensus/src/spendbundle_conditions.rs

```diff
@@ -108,6 +120,32 @@ mod tests {
             .fold(0, |sum, spend| sum + spend.create_coin.len());
         assert_eq!(create_coins, additions);
         assert_eq!(conditions.cost, cost);
+        // Generate a block with the same spend bundle and compare its cost
+        let program_spends: Vec<_> = bundle
+            .coin_spends
+            .iter()
+            .map(|coin_spend| {
+                (
+                    coin_spend.coin,
+                    &coin_spend.puzzle_reveal,
+                    &coin_spend.solution,
+                )
+            })
+            .collect();
+        let program = solution_generator(program_spends).expect("solution_generator failed");
+        let blocks: &[&[u8]] = &[];
+        let block_conds = run_block_generator2::<_, MempoolVisitor, _>(
+            &mut a,
+            program.as_slice(),
+            blocks,
+            11_000_000_000,
+            MEMPOOL_MODE,
+            &TEST_CONSTANTS,
+        )
+        .expect("run_block_generator2 failed");
+        const QUOTE_COST: u64 = 20;
+        // run_block_generator2 wraps the program in a quote
```

Comment:
this comment is not correct. `run_block_generator2` just *runs* the generator. it's `solution_generator()` that does the wrapping.  but executing the quote costs 20. However, you should also deduct the cost of the size of the wrapping, which I believe is 2 bytes